db/db_versioned.go

```diff
@@ -171,7 +205,10 @@ func (b *BoltDBVersioned) Delete(version uint64, ns string, key []byte) error {
 	}
```

Comment:
the behaviour when delete key does not exist is inconsistent with other kvstore impl, where doing nothing and return nil error